src/tools/spin/spinquic.cpp

```diff
@@ -731,103 +733,108 @@ main(int argc, char **argv)
     TryGetValue(argc, argv, "seed", &RngSeed);
     srand(RngSeed);
 
-    for (size_t i = 0; i < BufferCount; ++i) {
-        Buffers[i].Length = MaxBufferSizes[i]; // TODO - Randomize?
-        Buffers[i].Buffer = (uint8_t*)malloc(Buffers[i].Length);
-        EXIT_ON_NOT(Buffers[i].Buffer);
-    }
-
     SpinQuicWatchdog Watchdog((uint32_t)Settings.RunTimeMs + WATCHDOG_WIGGLE_ROOM);
```

Comment:
The threads are still configured to run `Settings.RunTimeMs` right? So if they do it more than once, the watchdog would fire, right?

Reply:
Good point. Moved it into the loop.

Reply:
I don't want the number of loops to multiple the total run time. I'd rather keep the watchdog where it is, and divide the time each loop runs.